client/landing/stepper/declarative-flow/site-migration-flow.ts

```diff
@@ -60,6 +88,7 @@ const siteMigration: Flow = {
 	},
 
 	useAssertConditions(): AssertConditionResult {
+		useMigrationFlowRevampRedirect();
```

Comment:
About where this logic should live, I added it here in the `useAssertConditions`, where we also have other redirects.